app/frontend/src/components/agriseafoodopscreening/Step2.vue

```diff
@@ -53,6 +53,13 @@
           :readonly="reviewMode"
           label="I have a plan to communicate physical distancing requirements to my employees (e.g. posting physical distancing signs in the accommodation area)."
         ></v-checkbox>
+
+        <v-checkbox
+          v-model="sharedSleepingProvidedAccomodations"
+          data-test="cb-form-sharedSleepingProvidedAccomodations"
```

Comment:
variables/fields should also have 2 m - sharedSleepingProvidedAccommodations

Reply:
Fixed